docs/literate.jl

```diff
@@ -40,7 +53,37 @@ function preprocess(content)
     # remove VSCode `##` block delimiter lines
     content = replace(content, r"^##$."ms => "")
 
-    return content
+    # adds the current version of the packages
+    append = """
+    # ### Package and system information
+    # #### Package version
+    # ```julia
+    $(chomp(replace(pkg_status, r"^"m => "# ")))
+    # ```
+    # #### Computer information
+    # ```julia
+    $(chomp(replace(sprint(InteractiveUtils.versioninfo), r"^"m => "# ")))
+    # ```
+    # #### Manifest
+    # Click [here to download the Manifest.toml]($(MANIFEST_OUT)) for this notebook's package environment
+    """
+    # This regex add "# " at the beginning of each line
+    # chomp removes trailing newlines
+
+    # The following gives a preview of the manifest
+    # embedded in a "spoiler", but this is unfortunately very buggy
```

Comment:
ah I was just thinking of changing it to that... what did you find buggy about it ?

Reply:
In the notebook it was not rendering well at all

Reply:
But this could be something we reserve to the markdown output...